shoop/configuration.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from __future__ import unicode_literals
+
+from shoop.core import cache
+from shoop.core.models import Shop, Configuration
+
+
+SHOP_CONFIGURATION_CACHE_PREFIX = "shop_configuration_%s"
+
+
+def get_cache_key(shop):
+    return SHOP_CONFIGURATION_CACHE_PREFIX % shop.pk if shop else ""
```

Comment:
Shouldn't it be `SHOP_CONFIGURATION_CACHE_PREFIX + str(shop.pk)` anyway? (it's a prefix, not a template)

i.e. with parenthesis too:
```python
return (SHOP_CONFIGURATION_CACHE_PREFIX + str(shop.pk)) if shop else ""
```